src/authorize.ts

```diff
@@ -56,7 +65,15 @@ export interface IEveryAuthOptions {
    * subsequent requests.  Usually extracts the value from the output of the authorization system, or can be a
```

Comment:
(on previous line) Both this and mapToTenantId are not "unique", so the word should probably be removed from both.

Reply:
fixed